react-challenge.html

```diff
@@ -67,21 +66,31 @@ <h1>Welcome to the Mood Meter App</h1>
       class MoodMeter extends React.Component{
         render(){
           return(
+            <div>
```

Comment:
✔️ remember, you can also use `React.Fragment` to avoid adding extra unnecessary nodes to your HTML :)